Multisig/UI/App/ContentView.swift

```diff
@@ -9,10 +9,11 @@
 import SwiftUI
 
 struct ContentView: View {
-
-    @State private var selection = 0
-    @State private var showsSafeInfo: Bool = false
-    @Environment(\.managedObjectContext) var context: CoreDataContext
+    @EnvironmentObject
+    var appState: App
```

Comment:
here we can either use the environment object, or initialize the selection object right in the property. And then pass this object as environment to the settings tab.